src/components/text-editor/test-setup/editor-state-builder.ts

```
@@ -0,0 +1,107 @@
+    return DOMParser.fromSchema(schema).parse(domNode);
+}
+
+/**
+ * Creates a simple empty document with optional text content.
+ *
+ * @param text - Optional text to include in the document
+ * @param schema - Optional custom schema (uses createTestSchema by default)
+ * @returns An EditorState with a simple document
+ */
+export function createDocumentWithText(
+    text: string = '',
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)